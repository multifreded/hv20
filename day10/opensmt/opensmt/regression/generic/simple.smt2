; This is an example

(set-info :bar "foo \"bar quux")
   (set-info :quux "bar quux frobboz")

(set-info :frobboz (foo :bar (quux bar)))

(set-logic AB_CD)
(declare-fun f () a)
(set-info :quux "bar quux 
frob\\boz\"")

(declare-fun _foo () b)

(declare-fun - () ..foo)

(declare-fun |bar "this is 


internal" quux| () ||)

(declare-fun q () (_ foo 1 2 3 4))

(declare-fun q1 () ((_ foo 1 2 3 4) (_ bar 4 3 2 1) (_ quux 1 2 3 4)))
