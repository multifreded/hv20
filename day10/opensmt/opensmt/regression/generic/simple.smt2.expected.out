(error "unknown logic AB_CD")

(error "Illegal command before set-logic: declare-fun")

(error "Illegal command before set-logic: declare-fun")

(error "Illegal command before set-logic: declare-fun")

(error "Illegal command before set-logic: declare-fun")

(error "Illegal command before set-logic: declare-fun")

(error "Illegal command before set-logic: declare-fun")

